conf/routes

```diff
@@ -55,6 +55,7 @@ POST   /api/v1/consent/dataSharing/resume    @controllers.ConsentController.resu
 # API - Upload
 POST   /api/v1/upload                  @controllers.UploadController.upload
 POST   /api/v1/upload/:id/complete     @controllers.UploadController.uploadComplete(id: String)
+GET    /api/v1/upload/:id/validationStatus  @controllers.UploadController.getValidationStatus(id: String)
```

Comment:
/api/v1/upload/:id/status? Is there any external notion of validation vs any other problem in the status of the upload that a user would be aware of?